### ISSUE TITLE
Internal server error /api/tasks?page_size=2000

### ISSUE THREAD

**Issue by azhavoro** [opencv/cvat#5491]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Current Behaviour

for some reson:
```python
File "/opt/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
  response = get_response(request)
File "/opt/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
  response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/opt/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
  return view_func(*args, **kwargs)
File "/opt/venv/lib/python3.8/site-packages/rest_framework/viewsets.py", line 125, in view
  return self.dispatch(request, *args, **kwargs)
File "/opt/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
  response = self.handle_exception(exc)
File "/opt/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
  self.raise_uncaught_exception(exc)
File "/opt/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
  raise exc
File "/opt/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
  response = handler(request, *args, **kwargs)
File "/opt/venv/lib/python3.8/site-packages/drf_spectacular/drainage.py", line 156, in wrapped_method
  return method(self, request, *args, **kwargs)
File "/opt/venv/lib/python3.8/site-packages/rest_framework/mixins.py", line 43, in list
  return self.get_paginated_response(serializer.data)
File "/opt/venv/lib/python3.8/site-packages/rest_framework/serializers.py", line 745, in data
  ret = super().data
File "/opt/venv/lib/python3.8/site-packages/rest_framework/serializers.py", line 246, in data
  self._data = self.to_representation(self.instance)
File "/opt/venv/lib/python3.8/site-packages/rest_framework/serializers.py", line 663, in to_representation
  return [
File "/opt/venv/lib/python3.8/site-packages/rest_framework/serializers.py", line 664, in <listcomp>
  self.child.to_representation(item) for item in iterable
File "/opt/venv/lib/python3.8/site-packages/rest_framework/serializers.py", line 515, in to_representation
  ret[field.field_name] = field.to_representation(attribute)
File "/opt/venv/lib/python3.8/site-packages/rest_framework/serializers.py", line 663, in to_representation
  return [
File "/opt/venv/lib/python3.8/site-packages/rest_framework/serializers.py", line 664, in <listcomp>
  self.child.to_representation(item) for item in iterable
File "/home/django/cvat/apps/engine/serializers.py", line 112, in to_representation
  label['svg'] = instance.skeleton.svg
File "/opt/venv/lib/python3.8/site-packages/django/db/models/fields/related_descriptors.py", line 421, in __get__
  raise self.RelatedObjectDoesNotExist(
cvat.apps.engine.models.Label.skeleton.RelatedObjectDoesNotExist: Label has no skeleton.
```

### Steps to Reproduce (for bugs)

1. Currently cannot reproduce locally

### Your Environment

- Git hash commit (`git log -1`): v2.3.0
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

**Comment by nmanovic** [comment]
@azhavoro , could you please add more details in the issue? How to reproduce it? A task or archive somewhere which will help to reproduce the issue.

**Comment by yasakova-anastasia** [comment]
I think the issue has been fixed by #5662 and #5700.
